<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="ATLANTISPC-274" name="Test A1" tests="4" failures="3" timestamp="9 Sep 2015 09:24:23 GMT" time="0.02" errors="0">
  <testcase name="testCaseA1" time="0.004" classname="com.qtpselenium.suiteA.TestCase_A1">
    <failure type="java.lang.AssertionError" message="expected [XYZ456] but found [XYZ]">
      <![CDATA[java.lang.AssertionError: expected [XYZ456] but found [XYZ]
	at com.qtpselenium.suiteA.TestCase_A1.testCaseA1(TestCase_A1.java:52)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- testCaseA1 -->
  <testcase name="testCaseA1" time="0.0" classname="com.qtpselenium.suiteA.TestCase_A1">
    <failure type="java.lang.AssertionError" message="expected [XYZ456] but found [XYZ]">
      <![CDATA[java.lang.AssertionError: expected [XYZ456] but found [XYZ]
	at com.qtpselenium.suiteA.TestCase_A1.testCaseA1(TestCase_A1.java:52)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- testCaseA1 -->
  <testcase name="testCaseA1" time="0.0" classname="com.qtpselenium.suiteA.TestCase_A1">
    <skipped/>
  </testcase> <!-- testCaseA1 -->
  <testcase name="testCaseA1" time="0.0" classname="com.qtpselenium.suiteA.TestCase_A1">
    <failure type="java.lang.AssertionError" message="expected [XYZ456] but found [XYZ]">
      <![CDATA[java.lang.AssertionError: expected [XYZ456] but found [XYZ]
	at com.qtpselenium.suiteA.TestCase_A1.testCaseA1(TestCase_A1.java:52)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- testCaseA1 -->
</testsuite> <!-- Test A1 -->
